recipes/pbipa/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pbipa" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "6d97a68a7209420183075915c6d7c3689efe77aae4833d1cb95decd82d44f4dd" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/PacificBiosciences/pbipa/releases/download/v{{ version }}/pbipa.tar.gz
+    sha256: {{ sha256 }}
+
+extra:
+  recipe-maintainers:
+    - pb-cdunn
+  skip-lints:
+    - should_be_noarch_generic # We use "skip", so this is a flawed lint-check.
+
+build:
+  number: 0
+  skip: True # [py27 or py36 or osx]
+
+requirements:
+  host:
```

Comment:
Please remove pcre and python. Also please add the following:

    build:
      - {{ compiler('c') }}

Then you can omit the skip-lints